tests/Keboola/BaseFunctionalTest.php

```diff
@@ -58,6 +58,27 @@ public function testRunThroughSSH(): void
         $this->assertEquals(0, $process->getExitCode(), $process->getErrorOutput());
     }
 
+    public function testLongTableName(): void
+    {
+        $this->initConfig(function ($config) {
+            if ($this->dataDir === ROOT_PATH . 'tests/data/functionalRow') {
+                $config['parameters']['dbName'] = 'reallyLongTableName' .
+                    'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx';
```

Comment:
tak mohl si tam dát nějaký https://www.php.net/manual/en/function.str-repeat.php ať to je vidět na první pohled kolik tam je znaků :D